sonic/client.py

```diff
@@ -222,7 +222,7 @@ def connect(self):
         return self.ping()
 
     def ping(self):
-        return self._execute_command("PING") == "PONG"
+        return self._execute_command("PING")
```

Comment:
this will override the usage of `self.raw` and force pythonify results always no? 
https://github.com/xmonader/python-sonic-client/blob/master/sonic/client.py#L348

Reply:
Should work in both cases now.

It now returns True to PONG, so that reuse() will successfully put the connection back into  the pool